### PR TITLE
Updated unit test for delete button.

### PR DESCRIPTION
**Pull request by AsmahanM**
A few things:
1. The default focus outlines were accidentally disabled, I've amended this now.
2. I've refactored the delete test- it now passes! 
3.  I added a class name to the delete button so that I could get my test to work.
4. Deleted the old unit tests for the delete button.

### CODE REVIEW

File: index-test.js [hunk 1]
```diff
@@ -39,33 +39,19 @@ test("form submits expected result", () => {
...
+document.querySelector('button#new-list').click(); //Click the TV button to generate new list
+const input = document.querySelector('input');
+input.value = "test"; //Add input value.
+document.querySelector('button#add-title').click(); //Click the 'plus' button to submit input.
+const todo = document.querySelector('.movie-name');
+todo.querySelector('button.delete').click(); //Click the delete button in the movie-name div to delete the input.
+const todoAfter = document.querySelector('.movie-name');
+equal(todoAfter, null, 'Item has been deleted.') //After the input has been deleted, the querySelecting movie-name should show up as null bc the item is deleted.
+todo.remove() //Clear the screen.
+document.querySelector('.list-container').remove();
```
Comment (mariaalouisaa) [inline index-test.js:133]: Well remembered ro reset the page!!

File: index-test.js [hunk 2]
```diff
@@ -39,33 +39,19 @@ test("form submits expected result", () => {
...
-  }, 5000);
-
-//Remove the delete button's class name. (Added in for test purposes)
+
+
+test("Delete button removes items from the list", () => {
+document.querySelector('button#new-list').click(); //Click the TV button to generate new list
+const input = document.querySelector('input');
+input.value = "test"; //Add input value.
+document.querySelector('button#add-title').click(); //Click the 'plus' button to submit input.
```
Comment (mariaalouisaa) [inline index-test.js:127]: Great commenting throughout, super clear!

File: index.js [hunk 3]
```diff
@@ -116,6 +116,7 @@ inputBtn.addEventListener('click', (e) => {
   deleteButton.ariaLabel = `Remove movie title from list`
   deleteButton.addEventListener('click', deleteItem)
   bingeWorthy.appendChild(deleteButton);
+  deleteButton.className = 'delete'
```
Comment (mariaalouisaa) [inline index.js]: Good spot!

**Review by mariaalouisaa — APPROVED** [review]
Well done for getting this done so quickly! Looks great and super clear to follow.

**Comment by mariaalouisaa** [comment]
There were quite a few merge conflicts that I fixed manually - hopefully I didn't break anything!!

Test looks great!

File: index.js (hunk 3 above)
Comment (PJSalter) [reply index.js]: This is great work Thank you ✨

File: index-test.js (hunk 1 above)
Comment (PJSalter) [reply index-test.js:133]: Good addition with the .remove() to clearing the screen as a restart, great work 🎉

File: index-test.js (hunk 2 above)
Comment (PJSalter) [reply index-test.js:127]: Really enjoyed reading the click on the plus button 🥳 ✨

**Pull request merged**